src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -334,11 +337,75 @@ class WelcomePage extends Disposable {
 				}
 			}
 		}));
+		this.createButtons();
 		this.createDropDown();
 		this.createWidePreviewToolTip();
 		this.createPreviewModal();
 	}
 
+	private createButtons(): void {
+		const dropdownButtonContainer = document.querySelector('#dropdown-btn-container') as HTMLElement;
```

Comment:
Using ids with very generic names like dropdown-btn-container seems like a recipe for disaster if you're querying the whole document. I'd suggest scoping down the search (which is a good idea anyways to reduce the overhead of having to traverse the entire DOM which shouldn't be necessary)